extensions/tmdb/package.json

```
@@ -0,0 +1,73 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "tmdb",
+  "title": "TMDB",
```

I think it would be better to not use an acronym in the name here, special for people that might not be familiar with TMDB acronym like myself. The Movie DB is also fine imo
```suggestion
  "title": "The Movie Database",
```